src/ServiceManager/Travely.ServiceManager.Service/Managers/ActivityManager.cs

```diff
@@ -1,36 +1,61 @@
-﻿using System;
+﻿using AutoMapper;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Travely.ServiceManager.Abstraction.Interfaces.UnitOfWorks;
+using ServiceManagerDb = Travely.ServiceManager.Abstraction.Models.Db;
 
 namespace Travely.ServiceManager.Service.Managers
 {
     public class ActivityManager : IActivityManager
     {
         protected readonly IUnitOfWork _unitOfWork;
+        protected readonly IMapper _mapper;
 
-        public ActivityManager(IUnitOfWork unitOfWork)
+        public ActivityManager(IUnitOfWork unitOfWork, IMapper mapper)
         {
             _unitOfWork = unitOfWork;
+            _mapper = mapper;
         }
-        public Task<Activity> CreateActivityAsync(Activity activity)
+        public async Task<Activity> CreateActivityAsync(Activity activity)
         {
-            throw new NotImplementedException();
+            var activityDTO = _mapper.Map<ServiceManagerDb.Activity>(activity);
```

Comment:
We create ServiceManagerDb.Activity twice - activityDTO & activityEntity. Probably, first one is not necessary. Please consider to remove it and work directly with activity.